aries-site/src/examples/templates/forms/utils/FormValidation.js

```diff
@@ -0,0 +1,88 @@
+export const emailMask = [
+  {
+    regexp: /^[\w\-_.]+$/,
+    placeholder: 'jane.smith',
+  },
+  { fixed: '@' },
+  {
+    regexp: /^[\w]+$/,
+    placeholder: 'hpe',
+  },
+  { fixed: '.' },
+  {
+    regexp: /^[\w]+$/,
+    placeholder: 'com',
+  },
+];
+
+export const phoneMask = [
+  { fixed: '(' },
+  {
+    length: 3,
+    regexp: /^[0-9]{1,3}$/,
+    placeholder: 'XXX',
+  },
+  { fixed: ')' },
+  { fixed: ' ' },
+  {
+    length: 3,
+    regexp: /^[0-9]{1,3}$/,
+    placeholder: 'XXX',
+  },
+  { fixed: '-' },
+  {
+    length: 4,
+    regexp: /^[0-9]{1,4}$/,
+    placeholder: 'XXXX',
+  },
+];
+
+export const emailValidation = [
+  {
+    regexp: new RegExp('[^@ \\t\\r\\n]+@'),
+    message: 'Enter a valid email address.',
+    status: 'error',
+  },
+  {
+    regexp: new RegExp('[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+'),
+    message: 'Enter a valid email address.',
+    status: 'error',
+  },
+  {
+    regexp: new RegExp('[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+'),
+    message: 'Enter a valid email address.',
+    status: 'error',
+  },
+];
+
+export const passwordRequirements = [
+    {
+      regexp: new RegExp('(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[#?!@$ %^&*-]).{8,}'),
+      message: 'Password requirements not met.',
+      status: 'error',
+    },
+  ];
+
+ export const passwordRulesStrong = [
+    {
+      regexp: new RegExp('(?=.*?[A-Z])'),
+      message: 'One uppercase letter',
+      status: 'error',
+    },
+    {
+      regexp: new RegExp('(?=.*?[a-z])'),
+      message: 'One lowercase letter',
+      status: 'error',
+    },
+    {
+      regexp: new RegExp('(?=.*?[#?!@$ %^&*-])'),
+      message: 'One special character',
+      status: 'error',
+    },
+    {
+      regexp: new RegExp('.{8,}'),
+      message: 'At least 8 characters',
+      status: 'error',
+    },
+  ];
+  
```

Comment:
Seems like it's flagging a missing "newline" at the end, we should add

Reply:
done